src/ops.rs

```diff
@@ -5,37 +5,71 @@ bi_enum! {
   /// A native operation on 60-bit integers.
```

Comment:
This enum needs a doc comment that explains the purpose of the `<...>S` variants.